client/src/components/App.jsx

```diff
@@ -9,6 +9,7 @@ import QuestionsList from './QuestionsAnswers/QuestionsList.jsx';
 
 const App = (props) => {
   const [currentProductId, changeCurrentProductId] = useState(21111);
```

Comment:
I think there might be a merge conflict here, as we had discussed yesterday moving towards using productId instead of currentProduct Id.